openmetadata-ui/src/main/resources/ui/src/utils/Alerts/AlertsUtil.tsx

```diff
@@ -141,3 +157,395 @@ export const getDisplayNameForEntities = (entity: string) => {
 
 export const EDIT_LINK_PATH = `/settings/notifications/edit-alert`;
 export const EDIT_DATA_INSIGHT_REPORT_PATH = `/settings/notifications/edit-data-insight-report`;
+
+const searchEntity = async (
```

Comment:
In some parts, we have this duplicated, so can remove the duplicate code and use this only?